tests/integration/test_charm.py

```diff
@@ -258,6 +268,11 @@ async def test_seldon_alert_rules(ops_test: OpsTest):
     client.delete(SELDON_DEPLOYMENT, name="seldon-model-1", namespace=namespace, grace_period=0)
     assert_deleted(client, SELDON_DEPLOYMENT, "seldon-model-1", namespace)
 
+    # cleanup Prometheus deployment
+    await ops_test.juju("remove-relation", f"{APP_NAME}", f"{prometheus}")
```

Comment:
do we really need to remove relation ? :)

Reply:
Just wanted complete clean up.
But it looks like remove-application does not work: remove-application call is blocked.

Reply:
```
Model            Controller          Cloud/Region        Version  SLA          Timestamp
test-charm-wj6q  microk8s-localhost  microk8s/localhost  2.9.43   unsupported  12:51:14-04:00

App                        Version  Status  Scale  Charm        Channel  Rev  Address        Exposed  Message
istio-pilot                         error     1/0  istio-pilot  edge     550  10.152.183.40  no       hook failed: "remove"
seldon-controller-manager           active      1  seldon-core             0  10.152.183.82  no       

Unit                          Workload  Agent  Address     Ports  Message
istio-pilot/0*                error     idle   10.1.59.79         hook failed: "remove"
seldon-controller-manager/0*  active    idle   10.1.59.77        
```

Reply:
well, then you found a bug in istio :D

Reply:
Bug filed.